src/components/Header/Header.tsx

```diff
@@ -3,11 +3,14 @@ import "./Header.scss";
 
 function Header() {
   return (
-    <div className="header">
-      <div className="header-title">To-do list</div>
+    <div className="header" id="header-height">
+      <div className="header-title" id="title-margin">
+        To-do list
+      </div>
       <img
         src={process.env.PUBLIC_URL + "/logo-coding-cerrado.png"}
         alt="logo"
+        id="header-height"
```

Comment:
I think it is possible to improve your solution without creating these new ids.

Reply:
Done